### PR TITLE
set geoengine-python/version as user agent for web requests

### PR DESCRIPTION
**Pull request by michaelmattig**

### CODE REVIEW

**Comment by michaelmattig** [comment]
[Geo Engine Python: Version als Agent an die Requests anhängen](https://trello.com/c/muEcIJKJ/880-geo-engine-python-version-als-agent-an-die-requests-anh%C3%A4ngen)

File: mypy.ini [hunk 1]
```diff
@@ -24,5 +24,8 @@ ignore_missing_imports = True
 [mypy-pandas.*]
 ignore_missing_imports = True
 
+[mypy-pkg_resources.*]
+ignore_missing_imports = True
+
```
Comment (ChristianBeilschmidt) [inline mypy.ini:29]: What about typings like https://pypi.org/project/types-pkg-resources/ ?

File: geoengine/__init__.py [hunk 2]
```diff
@@ -8,3 +11,12 @@
...
 from .colorizer import Colorizer
+
+DEFAULT_USER_AGENT = f'geoengine-python/{get_distribution("geoengine").version}'
+
+
+def default_user_agent(_name="python-requests"):
+    return DEFAULT_USER_AGENT
+
+
+utils.default_user_agent = default_user_agent
```
Comment (ChristianBeilschmidt) [inline geoengine/__init__.py:22]: Can we add a test for that or is that difficult?
Maybe with contains?

**Review by ChristianBeilschmidt — CHANGES_REQUESTED** [review]

File: mypy.ini (hunk 1 above)
Comment (michaelmattig) [reply mypy.ini]: 02165d7b2858ebbcb2bffae70657ed3d562ee4bb

File: geoengine/__init__.py (hunk 2 above)
Comment (michaelmattig) [reply geoengine/__init__.py:22]: 1ac93acd44c17e69741e5440a59e7b1ed5b756f7

**Review by ChristianBeilschmidt — APPROVED** [review]

**Comment by ChristianBeilschmidt** [comment]
bors merge

**Pull request merged**